lib/test.js

```diff
@@ -68,6 +68,24 @@ Test.prototype._assert = function (promise) {
 };
 
 Test.prototype._setAssertError = function (err) {
+	var data = err._avaThrowsHelperData;
+	if (data) {
+		console.error(
+			[
+				'Improper usage of t.throws detected at %s (%d:%d).',
+				'You should wrap the following expression in a function:',
+				'  %s',
+				'Like this:',
+				'  function() {\n    %s\n  }',
```

Comment:
I considered that.

But if the user is green enough that that they don't fully understand how `throw` works, I was concerned about piling on another abstraction they need to understand.